_includes/install/releasenotes/ee_install_21.md

```diff
@@ -40,4 +40,116 @@ After you get the Commerce software:
 
 ## Upgrade from an earlier version
```

Comment:
Do you need to keep this file since it refers to upgrading to version 2.1?

Reply:
Good catch @meker12. I'll remove it.

Reply:
Actually, the include is being referenced in the 2.2.1 release notes. I think the instructions apply to 2.2.1 instead of 2.1.x. I think I need to modify the include file so that it isn't specific to 2.1.

Reply:
Instead of modifying what is clearly supposed to be a 2.1-specific upgrade instruction set, I think I'll replace the reference to this include in the 2.2.1 release notes with a link to our general upgrade instructions: https://devdocs.magento.com/guides/v2.3/comp-mgr/cli/cli-upgrade.html.